leshan-core/src/test/java/org/eclipse/leshan/core/node/codec/LwM2mNodeEncoderTest.java

```diff
@@ -195,7 +195,7 @@ public void json_encode_device_object_instance() {
         b.append("{\"n\":\"9\",\"v\":100},");
         b.append("{\"n\":\"10\",\"v\":15},");
         b.append("{\"n\":\"11/0\",\"v\":0},");
-        b.append("{\"n\":\"13\",\"v\":1.367491215E9},");
+        b.append("{\"n\":\"13\",\"v\":1367491215},");
```

Comment:
Surprised to see this huge change in value? Are you sure that parsing here is working properly?

Reply:
I don't get it this is just 2 same way to write the "number".

1367491215 == 1.367491215E9

Reply:
Right. Sorry for confusion here.